openscenario/openscenario_interpreter/include/openscenario_interpreter/syntax/integer.hpp

```diff
@@ -35,6 +35,8 @@ struct Integer
 
   explicit Integer(const std::string &);
 
+  static auto infinity() noexcept -> Integer;
```

Comment:
As std::numeric_limits does not define a member function infinity for integer types, it is not possible to represent infinity with integer types. Please modify it to provide min and max.

Reply:
fixed in [`29bbac4` (#1316)](https://github.com/tier4/scenario_simulator_v2/pull/1316/commits/29bbac4258f0dd4c4c5381d959bd354ccc46d0dc)